package.json

```diff
@@ -4,13 +4,15 @@
   "description": "The service backend for the Philanthropy Data Commons",
   "main": "dist/index.js",
   "scripts": {
-    "build": "npm run cleanup && npm run build:node",
+    "build": "npm run cleanup && npm run build:node && npm run build:assets",
     "build:node": "tsc -p tsconfig.json",
+    "build:assets": "copyfiles -u 1 \"src/**/*.sql\" \"src/**/.keep\" dist",
     "cleanup": "rimraf dist",
     "lint": "eslint ./src --ext .ts",
-    "migrate": "ts-node -r dotenv/config src/scripts/migrate.ts",
+    "migrate": "npm run build && node -r dotenv/config dist/scripts/migrate.js",
```

Comment:
As annoying as it is to try to remember to build first and migrate second, I think having the `npm run build` here still means the `migrate` task depends on dev dependencies. So I could not run this `migrate` after `npm prune --omit=dev` because `tsc` wouldn't be available.

Reply:
Oh snap of course, silly me -- thanks.

I'll patch that up.

I'm gonna have to figure out if it's more common to have a `migrate:dev` or a `migrate:ci`, my initial instinct is it would be better to have migrate run using ts-node and migrate:ci run via node, that way nobody has to remember to build first.  Will spend a quick cycle thinking and throw out a proposal.

Reply:
Landed on migrate:dev for those who wanna skip having to run build first.

`migrate` and `start` are both assuming no dev dependencies.